app/serializers/application.js

```diff
@@ -5,4 +5,7 @@ export default DS.JSONAPISerializer.extend({
   keyForAttribute(attr) {
     return Ember.String.underscore(attr);
   },
+  keyForRelationship: function(key) {
+    return Ember.String.underscore(key);
+  },
```

Comment:
I think that's still the case on both sides, but would be nice to see if there's an option for it on the JSON API side.